python/lsst/ctrl/bps/transform.py

```diff
@@ -200,8 +200,8 @@ def create_init_workflow(config, qgraph, qgraph_gwfile):
     # Pick a node id for each task (not quantum!) to avoid reading the entire
     # quantum graph during the initialization stage.
     node_ids = []
-    for task in qgraph.iterTaskGraph():
-        task_def = qgraph.findTaskDefByLabel(task.label)
+    for task in qgraph.pipeline_graph.make_task_xgraph():
```

Comment:
```suggestion
    for task_label in qgraph.pipeline_graph.tasks:
```
No need to make a NetworkX graph here if you just want to iterate over task labels in order.